css/css-display/accessibility/display-contents-role-and-label.html

```diff
@@ -0,0 +1,290 @@
+          <th data-expectedrole="columnheader" data-expectedlabel="x" data-testname="th as child of tr with display: contents, within table with display: flex, has columnheader role" class="ex-role-and-label">x</th>
+          <th>x</th>
+        </tr>
+      </thead>
+      <tbody>
+        <tr style="display: contents;">
+          <td data-expectedrole="cell" data-expectedlabel="x" data-testname="td as child of tr with display: contents, within table with display: flex, has cell role" class="ex-role-and-label">x</td>
+          <td>x</td>
+        </tr>
+      </tbody>
+  </table>
+
+  <table role="table" style="display: flex; flex-flow: column nowrap;" data-expectedrole="table">
+    <thead>
+      <tr style="display: contents;" data-expectedrole="row" data-testname="tr with display: contents, within table with role=table with display: flex, has row role" class="ex-role">
+        <th data-expectedrole="columnheader" data-expectedlabel="x" data-testname="th as child of tr with display: contents, within table with role=table with display: flex, has columnheader role" class="ex-role-and-label">x</th>
+        <th>x</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr style="display: contents;">
+        <td data-expectedrole="cell" data-expectedlabel="x" data-testname="td as child of tr with display: contents, within table with role=table with display: flex, has cell role" class="ex-role-and-label">x</td>
+        <td>x</td>
+      </tr>
+    </tbody>
+  </table>
+
+  <table role="grid" style="display: flex; flex-flow: column nowrap;" data-expectedrole="grid">
+    <thead>
+      <tr style="display: contents;" data-expectedrole="row" data-testname="tr with display: contents, within table with role=grid with display: flex, has row role" class="ex-role">
+        <th data-expectedrole="columnheader" data-expectedlabel="x" data-testname="th as child of tr with display: contents, within table with role=grid with display: flex, has columnheader role" class="ex-role-and-label">x</th>
+        <th>x</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr style="display: contents;">
+        <td data-expectedrole="gridcell" data-expectedlabel="x" data-testname="td as child of tr with display: contents, within table with role=grid with display: flex, has gridcell role" class="ex-role-and-label">x</td>
+        <td>x</td>
+      </tr>
+    </tbody>
+  </table>
+
+  <!-- Display: Grid -->
+  <div style="display: grid; grid-template-columns: 1fr 100px; grid-gap: 1em;">
+      <h2>x</h2>
+      <ul style="display: contents" data-expectedrole="list" data-testname="ul with display: contents, as child of div with display: grid, has list role" class="ex-role">
+          <li>x</li>
+          <li data-expectedrole="listitem" data-expectedlabel="x" data-testname="listitem within ul with display: contents, as child of div with display: grid, has listitem role" class="ex-role-and-label">x</li>
+      </ul>
```

Comment:
Your editor seems to be 2-space indentations sometimes and 4 at other times. 

```suggestion
    <h2>x</h2>
    <ul style="display: contents" data-expectedrole="list" data-testname="ul with display: contents, as child of div with display: grid, has list role" class="ex-role">
      <li>x</li>
      <li data-expectedrole="listitem" data-expectedlabel="x" data-testname="listitem within ul with display: contents, as child of div with display: grid, has listitem role" class="ex-role-and-label">x</li>
    </ul>
```